ArmNavigation/seven_joint_arm_to_point_control/PathPlanner.py

```diff
@@ -0,0 +1,398 @@
+import math
```

Comment:
Please change the file name from "PathPlanner.py" to "rrt_star_seven_joint_arm.py"

Reply:
done!